src/Gui/DAGView/DAGView.cpp

```diff
@@ -80,14 +80,14 @@ void View::slotActiveDocument(const Document &documentIn)
 
 void View::slotDeleteDocument(const Document &documentIn)
 {
-  ModelMap::iterator it = modelMap.find(&documentIn);
+    ModelMap::const_iterator it = modelMap.find(&documentIn);
```

Comment:
There's a spacing problem here.

Reply:
Yep! The surrounding code is 2chr indented. In the pedantic spirit of not doing drive-by fixes I left it alone. It bit me twice before I realised.

Reply:
done